tests/integration/failPolicy_test.go

```diff
@@ -0,0 +1,79 @@
+package integration_test
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/rancher/lasso/pkg/client"
+	v3 "github.com/rancher/rancher/pkg/apis/cluster.cattle.io/v3"
+	provisioningv1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
+	"github.com/rancher/wrangler/pkg/gvk"
+	"github.com/rancher/wrangler/pkg/kubeconfig"
+	"github.com/rancher/wrangler/pkg/schemes"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/suite"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type FailurePolicySuite struct {
+	suite.Suite
+	clientFactory client.SharedClientFactory
+}
+
+// TestFailurePolicyTest should be run only when the webhook is not running.
```

Comment:
Nit: You could enforce this condition with a require statement somewhere which fails the test (with an explicit message) if any of the webhook pods are still up.

Totally optional though.

Reply:
Updated.